docl/resources/upload_docker_image.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
```

Comment:
in new repo there will be a `blueprint` directory where all other resources will be located. this file on the other hand will stay out side